src/form/Form.jsx

```diff
@@ -583,7 +585,9 @@ function Form({
           setFormSettings({
             redirectUrl: res.redirect_url,
             errorType: res.error_type,
-            autocomplete: res.autocomplete ? 'on' : 'off'
+            autocomplete: res.autocomplete ? 'on' : 'off',
+            formOff: Boolean(res.formOff),
```

Comment:
what is this formOff boolean? if the form is off no data will be sent back, i.e. {data: null}